src/coreclr/src/jit/indirectcalltransformer.cpp

```diff
@@ -116,6 +116,13 @@ class IndirectCallTransformer
                 transformer.Run();
                 count++;
             }
+
+            if (ContainsExpRuntimeLookup(stmt))
```

Comment:
Presumably only one of these `Contains...` can be true since we require the victim to be at or near the root of the tree. Would it make sense to early out when one fires?

Do we know for sure statement iteration is not messed up by these transformations?

We can exclude drilling into cases here by first looking at the various method flags again, right? Might save a bit of time. That is, if `OMF_HAS_FATPOINTER` is not set, no need to call `ContainsFatCalli`, and so on.

Reply:
>Would it make sense to early out when one fires?

It won't hurt, added.

I think if we are planning to add more and want to optimize than we probably should keep lists of statements that need transformations in Compiler class.